packages/net-security/sshguard/sshguard.exlib

```diff
@@ -15,31 +15,23 @@ LICENCES="
 "
 SLOT="0"
 
-MYOPTIONS="
-    ( providers:
-        hosts    [[ description = [ Use the hosts.allow/TCP wrappers backend ] ]]
-        iptables [[ description = [ Use the iptables/netfilter backend ] ]]
-    ) [[ number-selected = exactly-one ]]
-"
+MYOPTIONS=""
 
 DEPENDENCIES="
     build:
         sys-devel/bison
         sys-devel/flex
-    build+run:
-        providers:iptables? ( net-firewall/iptables )
+    recommendation:
+        net-firewall/iptables
```

Comment:
I take it it’s now automagic at runtime?

Reply:
Not really automagic, it all depends on which backend is selected in `sshguard.conf`. `sshg-fw-iptables` depends on `net-firewall/iptables`, `sshg-fw-nft-sets` depends on `net-firewall/nftables`, etc.

Reply:
I see, hence the `ls` in the news item, good.